data/data-pipeline/data_pipeline/etl/sources/dot_travel_composite/etl.py

```diff
@@ -0,0 +1,73 @@
+# pylint: disable=unsubscriptable-object
+# pylint: disable=unsupported-assignment-operation
+
+import pandas as pd
+import geopandas as gpd
+
+from data_pipeline.etl.base import ExtractTransformLoad, ValidGeoLevel
+from data_pipeline.utils import get_module_logger
+
+logger = get_module_logger(__name__)
+
+
+class TravelCompositeETL(ExtractTransformLoad):
+    """ETL class for the DOT Travel Disadvantage Dataset"""
+
+    NAME = "travel_composite"
+    SOURCE_URL = "https://www.transportation.gov/sites/dot.gov/files/Shapefile_and_Metadata.zip"
+    GEO_LEVEL = ValidGeoLevel.CENSUS_TRACT
+
+    # Output score variables (values set on datasets.yml) for linting purposes
+    TRAVEL_BURDEN_FIELD_NAME: str
+
+    def __init__(self):
+        # define the full path for the input CSV file
+        self.INPUT_SHP = (
+            self.get_tmp_path() / "DOT_Disadvantage_Layer_Final_April2022.shp"
+        )
+
+        # this is the main dataframe
+        self.df: pd.DataFrame
+
+        # Start dataset-specific vars here
+        ## Average of Transportation Indicator Percentiles (calculated)
+        ## Calculated: Average of (EPL_TCB+EPL_NWKI+EPL_NOVEH+EPL_COMMUTE) excluding NULLS
+        ## See metadata for more information
+        self.INPUT_TRAVEL_DISADVANTAGE_FIELD_NAME = "Transp_TH"
+        self.INPUT_GEOID_TRACT_FIELD_NAME = "FIPS"
+
+    def extract(self) -> None:
```

Comment:
nit: My guess is you can delete this method, since ti's what the method on the super class would do anyway.

Reply:
thank you!